4.0 connections enrichment (#607)
SimpleDataLabsInc/prophecy-docs: docs/administration/fabrics/prophecy-fabrics/connections/tableau.md

  - tableau
---

> A Tableau connection allows you to update data sources in Tableau projects. Use this integration to upload data from your pipeline as `Hyper` files (Tableau’s high-performance, in-memory database format).

Reviewer: ```suggestion
A Tableau connection allows you to update data sources in Tableau projects and to upload data from your pipeline as `Hyper` files (Tableau’s high-performance, in-memory database format).
```

Author: update data sources = upload data from pipeline as hyper files

so it's less `and` and more a=`by`... I will fix
